content/en/blog/README.md

```diff
@@ -0,0 +1,62 @@
+# Istio blog
+
+The Istio blog is owned by the [Steering committee](https://github.com/istio/community/blob/master/STEERING-COMMITTEE.md) and run by the [Editorial Team](#leadership).
+
+This section covers documentation, processes, and roles for the [Istio blog](https://istio.io/latest/blog/2020/).
+
+## Leadership 
+
+- **Editorial Lead:** [steering member company A], [steering member company B]
+- **Technical Editors:** [contributor], [contributor]
+- **Copy Editors:** [technical writer], [technical writer], [María Cruz](https://github.com/macruzbar)
+- **Blog Community Managers:**  [María Cruz](https://github.com/macruzbar), [independent contributor]
+
+## Contact
+
+- Slack: [#toc-steering-questions](https://istio.slack.com/archives/C018VK0T9JTT)
```

Comment:
It depends. What use case do you see that would require these submissions be made via email? Since we don't have embargoed content, we removed the private channel in favor of transparency.